src/plugins/WebOfTrust/util/SynchronousDelayedBackgroundJobFactory.java

```diff
@@ -0,0 +1,43 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * Background job factory implementation for {@link SynchronousDelayedBackgroundJob}s.
+ *
+ * @author bertm
+ * @see SynchronousDelayedBackgroundJob
+ */
+public class SynchronousDelayedBackgroundJobFactory
+        extends BackgroundJobFactoryBase
+        implements DelayedBackgroundJobFactory {
+    private final long defaultDelay;
+
+    /**
+     * Constructs a background job factory with given default delay.
+     * @param delay the default trigger aggregation delay
+     */
+    public SynchronousDelayedBackgroundJobFactory(long delay) {
+        defaultDelay = delay;
+    }
+
+    @Override
+    public SynchronousDelayedBackgroundJob newJob(Runnable job, String name) {
+        return newJob(job, name, defaultDelay);
+    }
+
+    /**
+     * Constructs a new {@link SynchronousDelayedBackgroundJob}. When this background job is
+     * {@link BackgroundJob#terminate() terminated}, the running job will be notified by
+     * interruption of its thread. Hence, the job implementer must take care not to swallow
+     * {@link InterruptedException}.
+     * @param job the job to run in the background
+     * @param name a human-readable name for the job
+     * @param delay the background job aggregation delay in milliseconds
+     * @see SynchronousDelayedBackgroundJob#SynchronousDelayedBackgroundJob(Runnable, String, long)
+     */
+    @Override
+    public SynchronousDelayedBackgroundJob newJob(Runnable job, String name, long delay) {
```

Comment:
This is also subject to the JavaDoc improment suggested here:
https://github.com/freenet/plugin-WebOfTrust/pull/26/files#r22342146

To not have to apply that change here as well, I would suggest to change the JavaDoc to:
```java
    /** Same as {@link #newJob(Runnable, String)} with the given delay instead of the default. */
```

The linked function already inherits the JavaDoc from the parent, so the suggest improvement can be placed there.